### DISCUSSION TITLE
Running SQL queries from the dashboard

### DISCUSSION THREAD

**Discussion by Nv7-GitHub** [pocketbase/pocketbase#1733] [Q&A] [answered]
Is it possible to run SQL queries form the dashboard? Would help a lot when trying to make queries.

**Comment by ganigeorgiev** [comment, marked as answer]
Currently this is not possible.

Aggregations and custom queries is in the [roadmap](https://github.com/orgs/pocketbase/projects/2) and will be eventually added in the next v0.13/v0.14 release as new "view" collection type.
